<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-13T02:26:56 WIB" name="Suite" finished-at="2024-09-13T02:26:57 WIB" duration-ms="718">
    <groups>
    </groups>
    <test started-at="2024-09-13T02:26:56 WIB" name="Facebook Login Test" finished-at="2024-09-13T02:26:57 WIB" duration-ms="648">
      <class name="futureTech9ExtentReport.LoginTest">
        <test-method is-config="true" signature="init()[pri:0, instance:futureTech9ExtentReport.LoginTest@35841320]" started-at="2024-09-13T02:26:56 WIB" name="init" finished-at="2024-09-13T02:26:56 WIB" duration-ms="246" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="LoginPageTest()[pri:0, instance:futureTech9ExtentReport.LoginTest@35841320]" started-at="2024-09-13T02:26:56 WIB" name="LoginPageTest" finished-at="2024-09-13T02:26:56 WIB" duration-ms="18" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Facebook HomePage] but found [Facebook Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Facebook HomePage] but found [Facebook Login]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at futureTech9ExtentReport.LoginTest.LoginPageTest(LoginTest.java:37)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTest -->
        <test-method is-config="true" signature="quit()[pri:0, instance:futureTech9ExtentReport.LoginTest@35841320]" started-at="2024-09-13T02:26:56 WIB" name="quit" finished-at="2024-09-13T02:26:57 WIB" duration-ms="340" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- futureTech9ExtentReport.LoginTest -->
    </test> <!-- Facebook Login Test -->
    <test started-at="2024-09-13T02:26:57 WIB" name="Facebook Registration Test" finished-at="2024-09-13T02:26:57 WIB" duration-ms="31">
      <class name="futureTech9ExtentReport.RegistrationTest">
        <test-method is-config="true" signature="init()[pri:0, instance:futureTech9ExtentReport.RegistrationTest@2584b82d]" started-at="2024-09-13T02:26:57 WIB" name="init" finished-at="2024-09-13T02:26:57 WIB" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="RegisTest()[pri:0, instance:futureTech9ExtentReport.RegistrationTest@2584b82d]" started-at="2024-09-13T02:26:57 WIB" name="RegisTest" finished-at="2024-09-13T02:26:57 WIB" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RegisTest -->
        <test-method is-config="true" signature="quit()[pri:0, instance:futureTech9ExtentReport.RegistrationTest@2584b82d]" started-at="2024-09-13T02:26:57 WIB" name="quit" finished-at="2024-09-13T02:26:57 WIB" duration-ms="18" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- futureTech9ExtentReport.RegistrationTest -->
    </test> <!-- Facebook Registration Test -->
  </suite> <!-- Suite -->
</testng-results>
